src/index.css

```diff
@@ -10,86 +10,4 @@
-	--weight-bold: 700;
-	--weight-extrabold: 800;
-	--weight-black: 900;
-	/* Color settings */
-	--color-foreground: hsla(0, 0%, 20%, 1); /* Default text color */
-	--color-background: hsla(0, 0%, 100%, 1); /* Background color */
-	--color-border: hsla(0, 0%, 80%, 1); /* Border color */
-	--color-card: hsla(0, 0%, 90%, 1); /* Card background color */
-	--color-helper: hsla(0, 0%, 60%, 1); /* Helper text color */
-	--color-accent: hsla(2, 84%, 56%, 1); /* Accent color (used for links, highlights) */
-	--color-error: hsla(0, 100%, 50%, 1); /* Error color */
-	--color-warning: hsla(44, 98%, 44%, 1); /* Warning color */
-	--color-success: hsla(120, 100%, 30%, 1); /* Success color */
-	--color-info: hsla(224, 100%, 60%, 1); /* Info color */
-	--color-white: hsla(0, 0%, 100%, 1); /* White color */
-	--color-black: hsla(0, 0%, 20%, 1); /* Black color */
-	--color-button: hsla(0, 0%, 0%, 0.15); /* Button color */
-	--color-shadow-light: hsla(0, 0%, 0%, 0.1); /* Light shadow color */
-	--color-shadow-heavy: hsla(0, 0%, 0%, 0.125); /* Heavy shadow color */
-	/* Spacing utilities */
-	--spacing-xs: calc(1 * var(--spacing-base)); /* Extra small spacing */
-	--spacing-sm: calc(2 * var(--spacing-base)); /* Small spacing */
-	--spacing-md: calc(3 * var(--spacing-base)); /* Medium spacing */
-	--spacing-lg: calc(5 * var(--spacing-base)); /* Large spacing */
-	--spacing-xl: calc(10 * var(--spacing-base)); /* Extra large spacing */
-	/* Border radius utilities */
-	--rounded-xs: calc(1 * var(--rounded-base)); /* Extra small border radius */
-	--rounded-sm: calc(2 * var(--rounded-base)); /* Small border radius */
-	--rounded-md: calc(4 * var(--rounded-base)); /* Medium border radius */
-	--rounded-lg: calc(8 * var(--rounded-base)); /* Large border radius */
-	--rounded-xl: calc(12 * var(--rounded-base)); /* Extra large border radius */
-	--rounded-full: 50%; /* Full circle border radius */
-	/* Border settings */
-	--border: var(--border-base) solid var(--color-border); /* Default border style */
-	--border-base: 1px; /* Base border width */
-	/* Shadow utilities */
-	--shadow-xs: 0 calc(0.5 * var(--shadow-base)) calc(0 * var(--shadow-base)) var(--color-shadow-light); /* Extra small shadow */
-	--shadow-sm: 0 calc(0.5 * var(--shadow-base)) calc(1 * var(--shadow-base)) var(--color-shadow-light); /* Small shadow */
-	--shadow-md: 0 calc(1 * var(--shadow-base)) calc(2 * var(--shadow-base)) var(--color-shadow-light); /* Medium shadow */
-	--shadow-lg: 0 calc(2 * var(--shadow-base)) calc(4 * var(--shadow-base)) var(--color-shadow-heavy); /* Large shadow */
-	--shadow-xl: 0 calc(4 * var(--shadow-base)) calc(8 * var(--shadow-base)) var(--color-shadow-heavy); /* Extra large shadow */
-	--shadow-2xl: 0 calc(8 * var(--shadow-base)) calc(12 * var(--shadow-base)) var(--color-shadow-heavy); /* 2x extra large shadow */
-	/* Accent color */
-	accent-color: var(--color-accent); /* Accent color used throughout the site */
-}
+@import './design-system.css' layer(design-system);
```

Comment:
The names have to match the first layer definition, or else the layers will not be applied in the expected order.

So, these names:
```css
@layer reset, fonts, design-system, base, pages, components, theme, print, accessibility;
```

Have to match these names:
```css
@import './basic-css-reset.css' layer(general-reset);
@import './basic-styling.css' layer(basic-styling);
@import './design-system.css' layer(design-system);
```